### PR TITLE
Workshops and courses

### PR DESCRIPTION
**Pull request by lkr-ind**
In this PR, I have added the details of the upcoming two Intel workshops, some details about the Nvidia DLI courses and how people can claim the free DLI course.

This can also be used as pages for advertising the relevant information. We can then use our newsletter to mention that we have created these pages and people should visit these pages for updated information.

This PR depends on #11. Please merge that one first.

### CODE REVIEW

File: docs/hpc/faq/python-multiprocessing.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+```bash
+#PBS -l select=1:ncpus=4:mem=16gb
+```
+
+In many cases, the multiprocessing package tries to utilize more memory (much higher than `16 GB` requested) or it tries to use more number of cores than what was actually requested through the scheduler. In many cases, the code starts using almost all the available cores on the processor. In such cases, the scheduler may kill the job sometimes. In other cases, the program may continue to run but the performance may degrade significantly.
+
+In almost all cases, the issue is not with the user's job. It is the configuration of the old PBS on our CX3 cluster.
+
+Quite recently, we have upgraded the PBS to a newer version on **CX3 phase 2** and this version does not cause any such issues. For more details on how to use **CX3 phase 2**, please see https://icl-rcs-user-guide.readthedocs.io/en/latest/hpc/pilot/cx3-phase2/.
```
Comment (davidhemp) [inline docs/hpc/faq/python-multiprocessing.md:19]:
```suggestion
For **CX3 phase 2** we have upgraded PBS to a newer version which does not cause any such issues. For more details on how to use **CX3 phase 2**, please see https://icl-rcs-user-guide.readthedocs.io/en/latest/hpc/pilot/cx3-phase2/.
```

File: docs/hpc/faq/python-multiprocessing.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Please note that the **CX3 phase 2** is connected with RDS and you can access all your data, environment and software installations as you would do it on **CX3 phase 1**. The only difference is you need to login to different login nodes as mentioned in the above link.
+
+If you are facing any issues with the multiprocessing package on **CX3 phase 1**, we would recommend you to use **CX3 phase 2**. If you are facing any issues with the multiprocessing package on **CX3 phase 2**, please let us know and we will try to help you out.
```
Comment (davidhemp) [inline docs/hpc/faq/python-multiprocessing.md:23]: Maybe link to the contact us page?

**Review by davidhemp — CHANGES_REQUESTED** [review]
Looks good, with a few language comments.

**Comment by lkr-ind** [comment]
Thanks @davidhemp  for your review. I have now made the changes.

**Review by davidhemp — APPROVED** [review]

**Pull request merged**